docs(migration): layouts/partial/favicons.html mirrors the one in the theme

aimeeu opened the pull request:
This should override the partial in the Docsy theme.
Move favicon.ico to static/favicons to mirror theme.

brian-armory commented:
overview and spinnaker directory done

brian-armory commented:
This doesn't include the EC2 "legacy" ones. Let's discuss a strategy for those on Friday before we bother.

aimeeu committed:
layouts/partial/favicons.html mirrors the one in the theme

This should override the partial in the Docsy theme.
Move favicon.ico to static/favicons to mirror theme.

aimeeu committed:
changed all icons to same href as shortcut icon
